contexts/security-v3-unstable.jsonld

```diff
@@ -321,6 +379,7 @@
     "signature": "https://w3id.org/security#signature",
     "signatureAlgorithm": "https://w3id.org/security#signingAlgorithm",
     "signatureValue": "https://w3id.org/security#signatureValue",
+    "proofValue": "https://w3id.org/security#proofValue",
```

Comment:
Move up into alpha order.

Reply:
won't fix without lint automation, @tplooker does your linter support sorting fields?

Reply:
You can use:
- https://marketplace.visualstudio.com/items?itemName=Tyriar.sort-lines if you use VSCode
- `:sort` command if you use Vim

Reply:
https://github.com/w3c-ccg/security-vocab/issues/83